c2rust-analyze/src/rewrite/expr/mir_op.rs

```diff
@@ -615,7 +619,13 @@ impl<'a, 'tcx> ExprRewriteVisitor<'a, 'tcx> {
                 }
                 _ => None,
             },
-            Ownership::Cell => None,
+            Ownership::Cell => match to.own {
+                Ownership::RawMut if !early => {
```

Comment:
This case should probably trigger on `Ownership::Raw` as well, since the conversion to `RawMut` can be used as the first step in `&Cell<T> -> *mut T -> *const T`.

Reply:
Is dd00d8d0 sufficient?

Reply:
Yes, that's what I was thinking